File: web3rpc/rpc-specs/code-samples/javascript/kaia/account/sign.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");

  const address = "0x487f2dfef230c2120b8cc55c5087b103146536ec";
  const message = "0xdeadbeaf";

  provider.kaia
    .sign(address, message, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/kaia/account/sign.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

address = "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b"
message = "0xdeadbeaf"

w3 = Web3(Web3.HTTPProvider(host))
kaia_response = w3.kaia.sign(address, message)

print(kaia_response)
